plugins/catalog-backend-module-bitbucket-cloud/config.d.ts

```diff
@@ -53,6 +55,32 @@ export interface Config {
                */
               projectKey?: RegExp;
             };
+            /**
+             * (Optional) TaskScheduleDefinition for the discovery.
+             * @visibility frontend
+             */
+            schedule?: {
```

Comment:
Is it possible to use `TaskScheduleDefinition` here instead of having to repeat this block? We don't really need this to be visible in the frontend anyway, do we?

Reply:
@Rugvip based on your comment I assume that these imports of types is ok at this file, right?

Reply:
I've replaced it with `TaskScheduleDefinition | string` now. Hope that is fine.